hydro_model_builder/model_generators/model_generator.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import json
+import geojson
+
+class model_generator(object):
+    def __init__(self, region_fn, options_fn):
+        self.region_fn = region_fn
+        self.options_fn = options_fn
+
+    def parse_config(self):
+        self.options = json.loads(self.option_fn)
```

Comment:
We probably should make `ModelGenrator` an abstract class, adding annotations `@abstractmethod` for methods to be implemented